### PR TITLE
Static pages for testing layout perf

### PR DESCRIPTION
**Pull request by tkadlec**
This adds some static pages so that we can get a really clear picture of where our heavy layout performance issues are coming from.

### CODE REVIEW

**Comment by tkadlec** [comment]
@schalkneethling @jwhitlock I think the pages are set, but first up I need to figure out how to get those pages to be accessible when the site is build. 

It _looks_ like I need to somehow get them to be pulled in when `collectstatic` runs, but I'm not 100% sure if that's true, or how to make that happen.

**Comment by tkadlec** [comment]
@schalkneethling I SWEAR I tried that URL before and it didn't work! Huh. 

Ok. I think we're all set then to merge these in.

File: kuma/static/layout-tests/no-inline-js.html [hunk 1]
```diff
@@ -0,0 +1,1438 @@
...
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <script>(function(d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
+
+  
+<link rel="preload" href="https://developer.mozilla.org/static/fonts/locales/ZillaSlab-Regular.subset.bbc33fb47cf6.woff2" as="font" type="font/woff2" crossorigin />
```
Comment (jwhitlock) [inline kuma/static/layout-tests/no-inline-js.html]: There's a risk that the hashed versions of these assets will not be available. We don't keep old versions of assets, but the CDN probably does. I'd say this is a low risk, but look for it in speedcurve. The same name without the hash will give the latest asset (``https://developer.mozilla.org/static/fonts/locales/ZillaSlab-Regular.subset.woff2``) with a 7-day cache timeout.

Comment (tkadlec) [reply kuma/static/layout-tests/no-inline-js.html]: Do you recommend dropping the has before deploying?

Comment (jwhitlock) [reply kuma/static/layout-tests/no-inline-js.html]: I'm not sure. Will the 7-day cache cause issues in speedcurve (continue using hashed, update if it breaks), or does it always request fresh (use unhashed and get latest)?

File: kuma/static/layout-tests/no-inline-js.html [hunk 2]
```diff
@@ -0,0 +1,1332 @@
...
+<!DOCTYPE html>
+<html lang="en-US" dir="ltr" class="no-js">
+<head prefix="og: http://ogp.me/ns#">
+  <meta charset="utf-8">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
```
Comment (jwhitlock) [inline kuma/static/layout-tests/no-inline-js.html]: Let's tell robots to ignore this duplicate content.

```suggestion
  <meta name="robots" content="noindex, nofollow">
```

File: kuma/static/layout-tests/no-js.html [hunk 3]
```diff
@@ -0,0 +1,1309 @@
...
+<!DOCTYPE html>
+<html lang="en-US" dir="ltr" class="no-js">
+<head prefix="og: http://ogp.me/ns#">
+  <meta charset="utf-8">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
```
Comment (jwhitlock) [inline kuma/static/layout-tests/no-js.html]: Let's tell robots to ignore this duplicate content.

```suggestion
  <meta name="robots" content="noindex, nofollow">
```

File: kuma/static/layout-tests/normal.html [hunk 4]
```diff
@@ -0,0 +1,1460 @@
...
+<html lang="en-US" dir="ltr" class="no-js">
+<head prefix="og: http://ogp.me/ns#">
+  <meta charset="utf-8">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <script>(function(d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
```
Comment (jwhitlock) [inline kuma/static/layout-tests/normal.html]: Let's tell robots to ignore this duplicate content.

```suggestion
  <meta name="robots" content="noindex, nofollow">
```

File: kuma/static/layout-tests/raw-html.html [hunk 5]
```diff
@@ -0,0 +1,1286 @@
...
+<!DOCTYPE html>
+<html lang="en-US" dir="ltr" class="no-js">
+<head prefix="og: http://ogp.me/ns#">
+  <meta charset="utf-8">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
```
Comment (jwhitlock) [inline kuma/static/layout-tests/raw-html.html]: Let's tell robots to ignore this duplicate content.

```suggestion
  <meta name="robots" content="noindex, nofollow">
```

File: kuma/static/layout-tests/unused-css.html [hunk 6]
```diff
@@ -0,0 +1,1452 @@
...
+<html lang="en-US" dir="ltr" class="no-js">
+<head prefix="og: http://ogp.me/ns#">
+  <meta charset="utf-8">
+
+  <meta http-equiv="X-UA-Compatible" content="IE=Edge">
+  <script>(function(d) { d.className = d.className.replace(/\bno-js/, ''); })(document.documentElement);</script>
+  <title>Array.prototype.forEach() - JavaScript | MDN</title>
+
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <meta name="robots" content="index, follow">
```
Comment (jwhitlock) [inline kuma/static/layout-tests/unused-css.html]: Let's tell robots to ignore this duplicate content.

```suggestion
  <meta name="robots" content="noindex, nofollow">
```

**Comment by jwhitlock** [comment]
Oh, and we need a bug number as well. @tkadlec, do you know if there is an existing performance bug in Bugzilla, or do we need a new one?

**Comment by tkadlec** [comment]
@jwhitlock I think we need a new one. It exists in sprints, but nowhere else that I know of.

File: kuma/static/layout-tests/no-inline-js.html (hunk 2 above)
Comment (tkadlec) [reply kuma/static/layout-tests/no-inline-js.html]: Good catch.

**Comment by tkadlec** [comment]
Changes made, using GH's beta "apply suggestion" thing which was pretty slick, other than needing separate commits for everything.

**Pull request merged**